guide/java/feeds.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Feeds
+layout: website-normal
+---
+
+<!-- TODO old, needs work (refactoring!) and use of java_link -->
+
+### Feeds
+
+A `Feed` is used to populate an entity's sensors. A variety of feed types are available.
```

Comment:
I think expand the explanation along the lines of 'A feed is a type of object Apache Brooklyn provides which can be used to populate an entity's sensors.' Then move 'A variety of feed types are available' to below the 'Types of Feed' header and / or expand what is being said, it currently feels like the prefix to a list.